_schemaV2.graphql

```diff
@@ -18522,6 +18524,11 @@ enum sort {
   DESC
 }
 
+enum SourceEnum {
```

Comment:
💅 Since this will be in the schema, I think we can prefix it with the model name, looks like it's the approach we use.

Reply:
Good point!